packages/paste-design-tokens/tokens/global/text-color.yml

```diff
@@ -16,7 +16,7 @@ props:
     value: "{!palette-gray-0}"
     comment: Inverse text color for dark backgrounds
   color-text-inverse-weak:
-    value: "{!palette-gray-60}"
+    value: "{!palette-gray-50}"
     comment: Weak inverse text color for dark backgrounds
```

Comment:
Are there any notes or gotchas related to this change we should add in this comment line?

Reply:
Do you mean whether we should change the comment to, e.g., "Weak inverse text color for dark backgrounds. Must pass AA color contrast with color-background-inverse"?

Or did you have something else in mind?

Reply:
Nothing specific in mind, just wanted to make sure to ask.  What you wrote above seems good to me!